src/main/java/io/thinkit/edc/client/connector/services/Service.java

```diff
@@ -0,0 +1,186 @@
+package io.thinkit.edc.client.connector.services;
+
+import static io.thinkit.edc.client.connector.utils.Constants.ID;
+import static io.thinkit.edc.client.connector.utils.JsonLdUtil.*;
+
+import com.apicatalog.jsonld.JsonLdError;
+import com.apicatalog.jsonld.document.JsonDocument;
+import io.thinkit.edc.client.connector.model.ApiErrorDetail;
+import io.thinkit.edc.client.connector.model.Result;
+import jakarta.json.JsonArray;
+import jakarta.json.JsonObject;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.function.UnaryOperator;
+
+public class Service {
```

Comment:
I'd give a different name, `Service` is pretty generic and it does not tell us enough about what it does.
Looking at the interface (`send`, `sendAsync`, functions to handle the response) it looks like a specialized http client, specialized for the management-api, so I would call it either `ManagementApiHttpClient` or `EdcHttpClient`